src/components/App.tsx

```diff
@@ -96,10 +99,12 @@ function useAboutMeRegister() {
 export default function App({ children }: { children: React.ReactNode }) {
   const pathname = usePathname();
   const router = useRouter();
+  const { getAccessToken } = useAuth();
 
   const navigateToNewChat = useCallback(() => {
     if (pathname !== "/") {
       router.push("/");
+      trackEvent('HL Chat New Conversation Started', {});
```

Comment:
When exactly does this fire right now?

Reply:
I think this event should explicitly fire when the user actually submits a message to an empty conversation. And then clicking "+ New" in nav is a separate event, because the conversation is only "started" when the user sends a message. Agree?

Reply:
I think it could be useful to check when they intend to create a new conversation when tapping on the create new chat button (tracked by this) and then send a message (which is tracked by a different event)

Reply:
Actually, getting rid of this